source/tutorials/first-steps/nix-language.md

```diff
@@ -1739,13 +1739,13 @@ For historical reasons, some of the functions in `pkgs.lib` are equivalent to [`
 So far we have only covered what we call *pure expressions*:
 declaring data and transforming it with functions.
 
-In practice, describing build tasks requires observing the outside world.
+In practice, describing derivations requires observing the outside world.
 
 There is only one impurity in the Nix language that is relevant here:
 reading files from the file system as *build inputs*
 
-Build inputs are files that build tasks refer to in order to describe how to derive new files.
-When run, a build task will only have access to explicitly declared build inputs.
+Build inputs are files that derivations refer to in order to describe how to derive new files.
+When run, a derivation will only have access to explicitly declared build inputs.
```

Comment:
I think here and below we should replace "run" with something else. Maybe "evaluated"?

Reply:
That will produce confusion with evaluating Nix expressions. The technically correct phrasing is ["running the `builder` executable of a derivation"](https://nixos.org/manual/nix/unstable/glossary#gloss-realise). That is usually condensed to "building a derivation", but I think it's somewhat misleading because it's both easy to confuse with "realising a derivation" (which can mean more things) and because "building" has the same semantic baggage as "build task".

Maybe we can in fact establish a more suitable term, but we shouldn't block this PR on that.